### DISCUSSION TITLE
PianoPlay - an OSMD-related project

### DISCUSSION THREAD

**Discussion by rvilarl** [opensheetmusicdisplay/opensheetmusicdisplay#989] [Show and tell]
Piano Play allows practicing MusicXML scores using a Piano  with a MIDI connector.
Enjoy it and have fun!

https://michaelecke.com/pianoplay/

Feedback/suggestions are welcome!

**Comment by hallvors** [comment]
Looks nice! Have you released the source code somewhere, or do you plan to open source it? :)

**Reply by rvilarl** [reply]
Thank you! For now I am not planning to open source it. 
But suggestions on interface/functionality are welcome.

**Comment by rvilarl** [comment]
Now open source in https://github.com/rvilarl/pianoplay
